test/testdata/infer/generics/variance_user.rb

```diff
@@ -73,7 +76,8 @@ def arrow_ok(arr); end
   # should fail: `Out` occurs in the contravariant position of Arrow, which is
   # in covariant position.
   sig {abstract.returns(Arrow[Out,Unit])}
-  def ret_arrow_pass; end
+  def ret_arrow_fail; end
+# ^^^^^^^^^^^^^^^^^^ error: `type_member` `Out` was defined as `:out` but is used in an `:in` context
```

Comment:
For return types, can we get the loc to point at the returns? Or is that hard?

Reply:
I'd like that too. I did a little digging on the method `Symbol`, but couldn't find where that location is stored.

Reply:
this is just a guess, but maybe there's some way to say `methodSym.data(ctx)->resultType->loc`? i.e., get the loc of the result type (again, just a hunch; this is non blocking)

Reply:
`loc` isn't property of the type.
We used to have multiple `loc`s on symbols but got rid of them to start using a single one.
Maybe now is a time to reconsider.